mpizzagalli/WoloxTraining
# Start of the proyect and creation of new articles

<h1> Summary
- Start and creation of the new proyect.
- Implemented feature to create new articles.

<h1> Test Plan
1. Go to http://localhost:3000/articles/new
2. Fill the form fields and save article.
3. Check the article is saved in the db.

<h1> Trello Card

https://trello.com/c/LVPfSJ2w/24-crear-blog-base
